src/main/jte/admin/users.jte

```diff
@@ -51,17 +51,23 @@
                         <td>${user.getUsername()}</td>
                         <td>${user.getEmail()}</td>
                         <td>${user.getUserId()}</td>
-                        <td>${user.isDisabled()}</td>
+                        @if(user.isDisabled())
+                            <td>${context.localize("administration.user.is_disabled.true")}</td>
+                        @else
+                            <td>${context.localize("administration.user.is_disabled.false")}</td>
+                        @endif
                         <td>
                             @if (user.isDisabled())
                                 @template.util.form(csrf = csrf, action = "/admin/users/" + user.getId() + "/reactivate", method = PUT, content = @`
-                                    <button type="submit" class="btn btn-ghost">${context.localize("administration.user.reactivate")}</button>
+                                    <button type="submit" class="btn btn-ghost" aria-label="${context.localize("administration.user.reactivate")}">${context.localize("administration.user.reactivate")}</button>
```

Comment:
This text is already the text of the button, I don't think repeating it is necessary.